analytical/templatetags/matomo.py

```diff
@@ -97,6 +97,19 @@ def render(self, context):
         if getattr(settings, 'MATOMO_DISABLE_COOKIES', False):
             commands.append(DISABLE_COOKIES_CODE)
 
+        if getattr(settings, "MATOMO_REQUIRE_CONSENT", False):
+            grant_class_name = settings.GRANT_CONSENT_TAG_CLASSNAME
```

Comment:
I'm not sure but this **might** cause issues for people using things like tailwindcss. Unless `document.getElementByClassName` can get elements despite the class attribute having multiple (space separated) values. This would need tested with that kind of set up